dse/Dockerfile-dse6.8.jdk11

```diff
@@ -169,10 +170,14 @@ COPY --chown=dse:root --from=dse-server-prep $DSE_AGENT_HOME $DSE_AGENT_HOME
 # Create folders
 RUN (for dir in /var/lib/cassandra \
                 /var/lib/spark \
+                /var/lib/spark/worker \
```

Comment:
I don't think these are required here. If anyone wants to use Spark, they have to create a mount /var/lib/spark that cass-operator then adds to the pod.

My idea was more like making entrypoint do the following:

```
if [ -d /var/lib/spark ]; then
  mkdir -p /var/lib/spark/worker
  mkdir -p /var/lib/spark/rdd
  mkdir -p /var/log/spark/worker
  mkdir -p /var/log/spark/master
fi
```

And not manually create /var/lib/spark at all. That way, user needs to add PVC (or EmptyVolume I guess, but that would feel weird) for the Spark data if they wish to use Spark.

Reply:
Because why would anyone store Spark files to the pod's image filesystem?

Reply:
Would we need to even create the spark sub directories? If `/var/lib/spark` is going to be a volume mount, there's no point in creating the sub directories in the Dockerfile as they will be masked by the volume mount, correct? So really, I think we can revert this part as only `/var/lib/spark` and `/var/log/spark` need to exist so that a volume can be mounted there later.

Reply:
No, we couldn't do it in the Dockerfile, but we can do it in the entrypoint (as previously). I think we should just not create /var/lib/spark at all perhaps? If it appears as mounted dir, then we create those subdirs.

Assuming there's even any need to create them. Why wouldn't Spark itself create them?

Reply:
Ok, I see. So if the parent directory exists, we assume it's a mounted volume and we can create the subdirs then. And remove the parent directory creation from the Dockerfile. So the only way the parent directories exist is if they are mounted, which wouldn't cause issue with a read-only root filesystem.